CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 ### Changed
+- Lookup groupId from device cache instead of directly from media stream when selecting input device.
```

Comment:
Can we backtick groupdId? Also, we need to update this to [Unreleased] Changed, currently, its under [2.13.0].